components/localized_input/localized_input.jsx

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import {intlShape} from 'react-intl';
+
+export default class LocalizedInput extends React.PureComponent {
```

Comment:
Accordingly to to usage, which is always pass a new object as placeholder, makes no sense to use a PureComponent here. It will rerender every single time and will check the properties.

Reply:
We should probably make that same change to the mobile app since I believe its version of this component is also a PureComponent